internal/services/errmessages/redactor.go

```diff
@@ -0,0 +1,59 @@
+// Package errmessages formats the error messages that are sent to the client.
+package errmessages
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"log/slog"
+
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+// RedactErrorInterceptor redacts some of the attached errors before sending it to the client.
+func RedactErrorInterceptor(ctx context.Context, req any, _ *grpc.UnaryServerInfo, handler grpc.UnaryHandler) (any, error) {
```

Comment:
I think it will be great for this package to have some tests too, no? This is probably something that we will keep anyway, so let’s do it properly first!